discord_api.py

```diff
@@ -74,18 +69,16 @@ def removeFooter(message):
 		del message_list[id_msg]
 		writeOnFile()
 		return ('{0.author.mention} Mensagem apagada!').format(message)
-	else:
-		return ('{0.author.mention} Não tens autorização para eliminar mensagens, mensagem não apagada').format(message)
+	return ('{0.author.mention} Não tens autorização para eliminar mensagens, mensagem não apagada').format(message)
 
 def showFooterMsgId(message):
 	if len(message_list) < 1:
 		return ('Não há mensagens inseridas no rodapé')
-	else:
-		resp = 'Lista de mensagens inseridas no rodapé: \n'
-		resp += '<autor> - <id> - <mensagem>\n'
-		for x in message_list:
-			resp += ('<@' + str(x[0].id) + '>' + ' - ' + str(message_list.index(x)+1) + ' - "' + x[1] + '"\n')
-		return resp.format(message)
+	resp = 'Lista de mensagens inseridas no rodapé: \n'
+	resp += '<autor> - <id> - <mensagem>\n'
+	for x in message_list:
```

Comment:
Let's use proper variable names. `x` How about `message`?

Reply:
I agree with you, so I've changed some variable names in addition to `x`.